inst/eDISH_app/ui.R

```diff
@@ -7,32 +8,34 @@ navbarPage("eDISH Shiny app", id = "inTabset",
                              wellPanel(
                                h3("Data upload"), 
                                fileInput("datafile", "Upload a csv or sas7bdat file",accept = c(".sas7bdat", ".csv"), multiple = TRUE),
-                              radioButtons("select_file","Select file for eDISH chart", choices = "No files available")
+                              radioButtons("select_file","Select file for eDISH chart", 
+                                           choiceNames = list(HTML("<p>Example data - <em style='color:green; font-size:12px;'>AdAm</em></p>")), 
+                                           choiceValues = "Example data") 
                              )
                       ),
                       column(6, 
                              fluidRow(
                                wellPanel( 
-                               h3("Data preview"), 
+                               uiOutput("datapreview_header"),
                                div(DT::dataTableOutput("data_preview"), style = "font-size: 75%")
                              )
                              ),
                              fluidRow(
                                br(),
-                               tags$style(type='text/css', '#detectStandard_msg {font-size:23px;}'),
-                               uiOutput("detectStandard_msg"),
-                               actionButton("view_chart","View Chart")
+                               tags$style(type='text/css', '#detectStandard_msg {font-size:23px;}')
                              )
                       )
                     )
            ),
            tabPanel(title = "Settings", id = "settings",
                     fluidPage(
+                      useShinyjs(),
                       renderSettingsUI("settingsUI")
                     )
            ),
            tabPanel(title = "Charts", value = "charts",
-                    eDISHOutput("chart")
+                      eDISHOutput("chart"),
+                      downloadButton("reportDL", "Export Chart", style = "position: relative; top: 440px;")  
```

Comment:
I think @jwildfire suggested placing the button at the bottom of page?

Reply:
This should be placing it at the bottom of the page -  I don't love using this css solution, but the html widget didn't seem to be going with the regular shiny bootstrap set up.

Reply:
@pburnsdata oh yes sorry! I was just commenting based off the "top".

Reply:
You're good!